packages/cli/src/lib/cmds/play.ts

```diff
@@ -39,36 +36,27 @@ export interface PlayArgs {
   * The Play class is the class that is used to communicate with the Google Play Store.
   */
 class Play {
+  private googlePlay?: GooglePlay;
```

Comment:
I liked when this was initialised in the constructor, as it prevents errors. 

Maybe we can pull the bit that loads the TWA Manifest to before this is initialised, so we can build GooglePlay and pass it on the constructor?

Reply:
Done.